test/eoftests/eoftests_runner.cpp

```diff
@@ -88,20 +87,16 @@ void from_json(const json::json& j, EOFTests& o)
 
 void from_json(const json::json& j, EOFTests::Case& o)
 {
-    std::string s{j.at("code").get<std::string>()};
+    std::string hex_str{j.at("code").get<std::string>()};
 
-    std::vector<uint8_t> vec;
-    for (std::size_t i = 2; i < s.length(); i += 2)
+    std::vector<uint8_t> bytes;
+
+    for (std::size_t i = 2; i < hex_str.size(); i += 2)
```

Comment:
Use `evmc::from_hex()` similar to https://github.com/ethereum/evmone/blob/095f83a3931c012af43e6580d188d4bb88af5d45/test/statetest/statetest_loader.cpp#L68